docs-md/README.md

```diff
@@ -24,6 +24,7 @@
   * [Push Notifications - Firebase](guides/push-notifications-firebase.md)
   * [Deep Links](guides/deep-links.md)
   * [Deploying and Updating](guides/deploying.md)
+  * [Splash Screens and Icons](guides/splash-screens-icons.md)
```

Comment:
put it here, since I think it's the best spot for now. Created a dedicated page not only for SEO/people searching for it, but to make it stand out more in the docs menu bar.